.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "editor.rulers": [80]
+  "editor.rulers": [80, 100]
```

Comment:
Is there a reason to keep 80?

Reply:
It's still useful for files other than *.cs* source files.